recipe/meta.yaml

```
@@ -1,176 +1,145 @@
-      summary: OpenBLAS headers and libraries for developing software that used OpenBLAS.
-      description: OpenBLAS is based on GotoBLAS2 1.13 BSD version.
-      doc_url: https://www.openblas.net/
-      doc_source_url: https://github.com/xianyi/OpenBLAS/blob/develop/README.md
-      dev_url: https://github.com/xianyi/OpenBLAS
-
-  # openblas meta-package. It is better to require openblas-devel or libopenblas
-  - name: openblas
-    requirements:
-      run:
-        - {{ pin_subpackage("libopenblas", exact=True) }}
-        - {{ pin_subpackage("openblas-devel", exact=True) }}
-        - blas=*=openblas
-        - {{ pin_subpackage("nomkl", exact=True) }}
-
-    about:
-      home: https://www.openblas.net/
-      license: BSD-3-Clause
-      license_file:
-        - LICENSE
-        - lapack-netlib/LICENSE
-      license_family: BSD
-      summary: An optimized BLAS library
-      description: OpenBLAS is based on GotoBLAS2 1.13 BSD version.
-      doc_url: https://www.openblas.net/
-      doc_source_url: https://github.com/xianyi/OpenBLAS/blob/develop/README.md
-      dev_url: https://github.com/xianyi/OpenBLAS
-
-  # mutex package to keep only one blas implementation in a given env
-  # Since this package has a non-unique name, blas-1.0-openblas.conda, it is omitted from future updates as they cause issues
-  # with overwrites on the PBP. If an update to this package is needed in the future, the name should have a hash added
-  # so that it becomes unique per publish.
-#  - name: blas
-#    version: 1.0
-#    build:
-#      string: openblas
-#      # track_features doesn't really track anything anymore (blas metapackage
-#      # dependencies do the same job better). This is still here, though, as it
-#      # effectively "weighs down" nomkl packages, allowing mkl to take
-#      # precedence when defaults is the top channel priority.
-#      track_features:
-#        - nomkl
-
-  - name: nomkl
-    version: 3.0
+        - {{ pin_subpackage("libopenblas" ~ name_suffix, exact=True) }}
+        - nomkl
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libopenblas{{ SYMBOLSUFFIX }}.a                       # [not win]
+        - test -f ${PREFIX}/lib/libopenblas{{ SYMBOLSUFFIX }}{{ SHLIB_EXT }}          # [not win]
+        - if not exist %PREFIX%\\Library\\lib\\openblas{{ SYMBOLSUFFIX }}.lib exit 1  # [win]
+  # Maintain compatibility with old package names
+  - name: openblas-devel{{ name_suffix }}
     build:
-      string: "0"
-      number: 0
+      string: openmp_h{{ PKG_HASH }}_{{ build_number }}  # [USE_OPENMP == 1]
+      string: pthreads_h{{ PKG_HASH }}_{{ build_number }}  # [USE_OPENMP == 0]
+      track_features:
+        - openblas_threading_openmp     # [(not osx) and USE_OPENMP == 1]
     requirements:
       run:
-        - blas * openblas
-    about:
-      license: BSD-3-clause
+        - {{ pin_subpackage("openblas" ~ name_suffix, exact=True) }}
+        - nomkl
+    run_exports:
+      - blas * openblas
```

```suggestion
    run_exports:
      - blas * openblas
      - {{ pin_subpackage("libopenblas" ~ name_suffix) }}
```
I think this might also need `libopenblas`. Since these come out as a weak export, we'd want inclusions of either `openblas` or `openblas-devel` to produce the same runtime requirements in downstream packages.